modules/rostests/rosautotest/main.cpp

```diff
@@ -87,6 +89,14 @@ extern "C" int
 wmain(int argc, wchar_t* argv[])
 {
     int ReturnValue = 1;
+    DWORD TestStartTime, TestEndTime;
+
+    GetModuleFileNameW(NULL, TestName, _countof(TestName));
+//    printf("Full TestName is '%S'\n", TestName);
+    WCHAR* Name = wcsrchr(TestName, '\\');
+    if (Name)
+        memmove(TestName, Name + 1, _countof(TestName) * sizeof(WCHAR));
```

Comment:
```suggestion
        memmove(TestName, Name + 1, (wcslen(Name + 1) + 1) * sizeof(WCHAR));
```
explanation: wcslen(Name+1) for the length in chars of the nul-terminated substring starting at Name+1 (after the backslash), +1 for counting the nul Terminator, and * sizeof(wchar) to convert to bytes.